### PR TITLE
Adds Waiting List API V2 tutorial with it's dependencies

### PR DESCRIPTION
**Pull request by amatiszcr**

### CODE REVIEW

File: data/array/first item getter (with not found)/e408f545-5648-47e7-9392-6c39c0dd14d1.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
+  "name": "data/array/First item getter (with not found)",
```
Comment (lvass74) [inline data/array/first item getter (with not found)/e408f545-5648-47e7-9392-6c39c0dd14d1.json:2]: Better to extend the current "data/array/First item getter" with not found output port.

File: data/array/type tester fork/19aa85fb-32fd-4877-a6c8-96aa1865e89e.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      "498fba4a-b825-49aa-a999-b46f46c9bc46": {
+        "type": "any",
+        "name": "data",
+        "description": "Receives the data to be forwarded."
+      }
+    },
+    "outputs": {
+      "54739c09-5a28-4237-9e38-38e8aed6b189": {
+        "type": "any[]",
+        "name": "on array",
```
Comment (lvass74) [inline data/array/type tester fork/19aa85fb-32fd-4877-a6c8-96aa1865e89e.json:14]: Other typed fork keep the output ports named as `on true` and `on false`. For consistency we could use this same naming scheme here too.

File: tutorials/waiting list api v2/common/at records to subscription converter/ed3b57f7-f7db-418e-a564-ce43841d6146.json [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+          ],
+          "name": "first item getter (with not found)",
+          "prototype": "e408f545-5648-47e7-9392-6c39c0dd14d1"
+        },
+        "fe0ae2be-d010-47f2-b6cd-b12522017b9d": {
+          "coordinates": [
+            23.5,
+            91.25
+          ],
+          "name": "first item getter (with not found) B",
```
Comment (lvass74) [inline tutorials/waiting list api v2/common/at records to subscription converter/ed3b57f7-f7db-418e-a564-ce43841d6146.json:48]: better node name?

**Review by lvass74 — CHANGES_REQUESTED** [review]
Review still in progress...

File: tutorials/waiting list api v2/resource/subscribe/token setter/f0b00122-f92d-4d12-8453-e3d083fad9ba.json [hunk 4]
```diff
@@ -0,0 +1,76 @@
+{
+  "name": "tutorials/Waiting list API V2/resource/subscribe/token setter",
```
Comment (lvass74) [inline tutorials/waiting list api v2/resource/subscribe/token setter/f0b00122-f92d-4d12-8453-e3d083fad9ba.json:2]: From the namespace of this prototype I expected it to be internal but it is public (reusable). Is this intentional?

File: tutorials/waiting list api v2/resource/validate/3af9300a-c73a-4c13-9683-4e6983bec65e.json [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+  "interface": {
+    "inputs": {
+      "dae7b830-396f-479a-9331-55cc629af727": {
+        "type": "{\n  \"serverId\": string,\n  \"port\": number\n}",
+        "name": "API params"
+      },
+      "929e8d94-bfe4-4701-9e7f-b7a46b6ad482": {
+        "type": "{\n  \"apiKey\": string,\n  \"baseId\": string,\n  \"tableName\": string\n}",
+        "name": "AT params",
+        "description": "[Inherited from port `params` of `save to AirTable`] \n[Inherited from port `params` of `record inserter`] \nReceives the authentication & table parameters required to construct the request.\n\nAPI Key: In AirTable, go to 'Help?' and click on 'API Documentation'. Scroll to the \"Authorization\" section to retrieve your API key.\n\nBase ID: In AirTable, go to 'Help?' and click on 'API Documentation'. Find the ID of the base in the introduction.\n\nTable Name: Use the table name located on the tab of the spreadsheet.\n\nExample:\n```json\n{\n  \"apiKey\": \"key...\",\n  \"baseId\": \"app...\",\n  \"tableName\": \"...\"\n}\n```"
```
Comment (lvass74) [inline tutorials/waiting list api v2/resource/validate/3af9300a-c73a-4c13-9683-4e6983bec65e.json:12]: Review or clear

File: tutorials/waiting list api v2/resource/validate/is validated fork/query airtable/filter builder/8433bdf5-e2b8-4aff-a9ae-6fab14e404ee.json [hunk 6]
```diff
@@ -0,0 +1,80 @@
+{
+  "name": "tutorials/Waiting list API V2/resource/validate/is validated fork/query airtable/filter builder",
+  "interface": {
+    "inputs": {
+      "d8251ef2-8ae5-4752-8941-b6ba737a33a3": {
+        "type": "typeof `value` of `builder (key & value)`",
+        "name": "token",
+        "description": "[Inherited from port `value` of `builder (key & value)`] \nReceives the value to assign to all items.\n\nExample:\n1"
```
Comment (lvass74) [inline tutorials/waiting list api v2/resource/validate/is validated fork/query airtable/filter builder/8433bdf5-e2b8-4aff-a9ae-6fab14e404ee.json:8]: Review or clear

File: tutorials/waiting list api v2/resource/validate/is validated fork/query airtable/filter builder/8433bdf5-e2b8-4aff-a9ae-6fab14e404ee.json [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+      }
+    },
+    "outputs": {
+      "5be45501-f8e0-4121-b092-153055250566": {
+        "type": "typeof `filled` of `custom template filler`",
+        "name": "query",
+        "description": "[Inherited from port `filled` of `custom template filler`] \nSends the template string with replaced parameters.\n\nExample:\n\"Hello world!\""
```
Comment (lvass74) [inline tutorials/waiting list api v2/resource/validate/is validated fork/query airtable/filter builder/8433bdf5-e2b8-4aff-a9ae-6fab14e404ee.json:15]: Review or clear

File: tutorials/waiting list api v2/resource/validate/response builder/6405424b-d890-47a2-a565-52554ab2cb7b.json [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+            "e3569ca8-4547-41da-be08-b3fd185a8b72": {
+              "0b28df10-b056-4666-ae14-2357d747f7b9": {}
+            }
+          }
+        },
+        "outputs": {
+          "18e7ed6a-dd64-412c-8331-94eb06e94d94": {
+            "7144327c-4e98-4cd1-81fa-b02f89356368": {
+              "cd2cfdb3-e99f-4836-b743-42a5834316f7": {
+                "description": "undefined\nundefined"
```
Comment (lvass74) [inline tutorials/waiting list api v2/resource/validate/response builder/6405424b-d890-47a2-a565-52554ab2cb7b.json:158]: Clear undefined values

**Review by lvass74 — CHANGES_REQUESTED** [review]

File: tutorials/waiting list api v2/common/is validated fork/519c16b7-898b-49eb-bff4-87c52e1cc55b.json [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+      "e7587b53-b89e-42de-b35e-f231363bc174": {
+        "type": "{\n\"email\": string,\noptional \"validated\": boolean,\noptional \"validation_token\": string\n}",
+        "name": "subscription",
+        "description": ""
+      }
+    },
+    "outputs": {
+      "0e5301d7-cb30-4ef0-829e-0f6f7b4c3fe5": {
+        "type": "{\n\"email\": string,\n\"validated\": false,\noptional \"validation_token\": string\n}",
+        "name": "not validated",
```
[the remark this answers is not in the document]
Comment (amatiszcr) [reply tutorials/waiting list api v2/common/is validated fork/519c16b7-898b-49eb-bff4-87c52e1cc55b.json:14]: As above, but will add "on"

**Review by lvass74 — APPROVED** [review]

**Pull request merged**